src/core/mod.rs

```diff
@@ -25,58 +39,39 @@ impl ClientState for WalkerCache {
     type ReadDirState = Self;
 }
 
+/// Specialized type for directory entries with this module's walker cache.
 type Entry = DirEntry<WalkerCache>;
 
-pub mod dir_rm;
-pub mod dir_stats;
-mod matching_state;
-pub use matching_state::MatchingState;
+mod dir_rm;
+pub use dir_rm::dir_rm_parallel;
```

Comment:
why do we mix mod's and use's.

Reply:
See [above](https://github.com/Kacper0510/project_cleaner/pull/7#discussion_r1727393117)